samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py

```diff
@@ -1,27 +1,30 @@
 #!/usr/bin/env python3
 
 
+import ast
+import json
+import random
 import kfp
 from kfp import components
 from kfp import dsl
 
-sagemaker_hpo_op = components.load_component_from_file(
-    "../../../../components/aws/sagemaker/hyperparameter_tuning/component.yaml"
+sagemaker_hpo_op = components.load_component_from_url(
+    "https://raw.githubusercontent.com/kubeflow/pipelines/f162b248f42f0a9a8999b8e709fb05fa05b40b8d/components/aws/sagemaker/hyperparameter_tuning/component.yaml"
 )
-sagemaker_process_op = components.load_component_from_file(
-    "../../../../components/aws/sagemaker/process/component.yaml"
+sagemaker_process_op = components.load_component_from_url(
+    "https://raw.githubusercontent.com/kubeflow/pipelines/f162b248f42f0a9a8999b8e709fb05fa05b40b8d/components/aws/sagemaker/process/component.yaml"
 )
-sagemaker_train_op = components.load_component_from_file(
-    "../../../../components/aws/sagemaker/train/component.yaml"
+sagemaker_TrainingJob_op = components.load_component_from_url(
+    "https://raw.githubusercontent.com/kubeflow/pipelines/58d22d4ba91b42e9c63ce88c5e349eadbe26d33b/components/aws/sagemaker/TrainingJob/component.yaml"
 )
-sagemaker_model_op = components.load_component_from_file(
-    "../../../../components/aws/sagemaker/model/component.yaml"
+sagemaker_model_op = components.load_component_from_url(
+    "https://raw.githubusercontent.com/kubeflow/pipelines/f162b248f42f0a9a8999b8e709fb05fa05b40b8d/components/aws/sagemaker/model/component.yaml"
 )
-sagemaker_deploy_op = components.load_component_from_file(
-    "../../../../components/aws/sagemaker/deploy/component.yaml"
+sagemaker_deploy_op = components.load_component_from_url(
+    "https://raw.githubusercontent.com/kubeflow/pipelines/f162b248f42f0a9a8999b8e709fb05fa05b40b8d/components/aws/sagemaker/deploy/component.yaml"
 )
-sagemaker_batch_transform_op = components.load_component_from_file(
-    "../../../../components/aws/sagemaker/batch_transform/component.yaml"
+sagemaker_batch_transform_op = components.load_component_from_url(
```

Comment:
use relative paths as before and add a git clone command if needed so that we dont have to update this for each release